aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -461,6 +462,11 @@ module aptos_framework::stake {
         move_to(account, owner_cap);
     }
 
+    /// Destroy `owner_cap`.
+    public fun destroy_owner_cap(owner_cap: OwnerCapability) {
```

Comment:
how does this function relate to this change?

Reply:
Not really related. Just wanted to add this so there's a way to destroy owner cap for StakePool that has been dissolved.